opentok-angular.js

```diff
@@ -195,6 +196,11 @@ ng.module('opentok', [])
               ng.element(element).addClass('allowed');
               scope.$emit('otAccessAllowed');
             },
+            destroyed : function (e) {
+              if (scope.streamForCanvas){
+                scope.streamForCanvas.forEach(track => track.stop())
+              }
```

Comment:
Is this something that should be done at the OT level?

Reply:
I don't think so. We're making a GDM call at the application level.

Reply:
Hypothetically, a customer might pass in a custom video source and want to keep it open after the publisher is destroyed.